system/include/emscripten/em_js.h

```diff
@@ -7,6 +7,8 @@
 
 #pragma once
 
+#include <emscripten/em_macros.h>
```

Comment:
Is this related?

Reply:
Yes, its need to include `em_js.h` without including the master `emscripten.h`.. which I do in the new test.